app/src/main/java/com/example/concordia_campus_guide/Fragments/PathInfoCardFragment/PathInfoCardFragment.java

```diff
@@ -252,6 +245,12 @@ public void setOutdoorTopCard() {
                             setDividerTextView(layout, layoutParams);
                     }
                     break;
+//                    TODO: Shuttle constant has not been created yet
```

Comment:
it has been created no? In Caren's PR it has.

Reply:
and that PR was merged right?

Reply:
the shuttle connection with the path activity sint implememted yet

Reply:
![image](https://user-images.githubusercontent.com/35609369/77877860-f13e4300-7224-11ea-897a-7ad68df98cee.png)